gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/PasswordPolicyResource.java

```diff
@@ -107,4 +109,30 @@ public void update(
                 .subscribe(response::resume, response::resume);
     }
 
+    @DELETE
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Delete a password policy",
+            operationId = "deletePasswordPolicy",
+            description = "User must have the DOMAIN_SETTINGS[UPDATE] permission on the specified domain " +
+                    "or DOMAIN_SETTINGS[UPDATE] permission on the specified environment " +
```

Comment:
**question**: shouldn't be [DELETE]?

Reply:
I asked the question to myself. I put UPDATE because even if it is a REST resource deletion, we are updating the domain settings (how the password are validated), I also check where this DOMAIN_SETTINGS permission was used and we only use the READ and UPDATE ACLs never the DELETE one